pydantic/fields.py

```diff
@@ -340,13 +340,35 @@ def __repr_args__(self) -> ReprArgs:
                     yield s, value
 
 
-def Field(
+class AliasPath:
+    def __init__(self, first_arg: str, *args: str | int) -> None:
+        self.paths = [first_arg] + list(args)
+
+    def convert_to_aliases(self) -> list[str | int]:
+        return self.paths
+
+
+class AliasChoices:
+    def __init__(self, *args: str | AliasPath) -> None:
+        self.choices = args
+
+    def convert_to_aliases(self) -> list[list[str | int]]:
+        aliases: list[list[str | int]] = []
+        for c in self.choices:
+            if isinstance(c, AliasPath):
+                aliases.append(c.convert_to_aliases())
+            else:
+                aliases.append([c])
+        return aliases
+
+
+def Field(  # noqa C901
     default: Any = Undefined,
     *,
     default_factory: typing.Callable[[], Any] | None = None,
     alias: str | None = None,
     alias_priority: int | None = None,
-    validation_alias: str | list[str | int] | list[list[str | int]] | None = None,
+    validation_alias: str | AliasChoices | AliasPath | None = None,
```

Comment:
```suggestion
    validation_alias: str | AliasPath | AliasChoices | None = None,
```
this order seems more natural to me but I'm okay either way.

Reply:
Done